src/styles/variables.scss

```diff
@@ -0,0 +1,13 @@
+$header-color-active: #00acdc;
+$height-header: 60px;
+$margin-header: 50px;
+$bg-color-header: #fff;
+
+$secondary-color-gray: #616070;
```

Comment:
It is not necessary to specify the color itself in the name, it can change during the development process and it will have to be corrected in the entire code.